Resources/Prototypes/_Goobstation/Entities/Clothing/Neck/specific.yml

```
@@ -0,0 +1,32 @@
+  parent: ClothingNeckBase
+  id: SurveillanceWirelessCameraBodySecurity
+  name: security bodycam
+  description: A body mounted camera. It's watching you.
+  components:
+  - type: DeviceNetwork
+    deviceNetId: Wired # :clueless:
+    receiveFrequencyId: SurveillanceCameraSecurity
+    transmitFrequencyId: SurveillanceCamera
+  - type: SurveillanceCamera
+    networkSet: true
+    nameSet: True
```

yamllint (1.35.1)

[warning] truthy value should be one of [false, true]

(truthy)